argocd-helm-charts/fluent-bit/templates/netpol-fluentbit.yaml

```diff
@@ -0,0 +1,36 @@
+{{ if .Values.networkpolicies }}
```

Comment:
@guilherme-santos03 this flag needs to be present on the values.yaml for fluent bit chart to enable/disable this template.